packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -186,7 +184,7 @@ export const BootDataProvider = ({
   const updatedAtActive = user ? dataUpdatedAt : null;
   const updateQueryCache = useCallback(
     (updatedBootData: Partial<BootCacheData>, update = true) => {
-      const cachedData = getCachedOrNull() || {};
+      const cachedData = getCachedBootOrNull() ?? {};
```

Comment:
This might result to error. It will not fallback if the value is `null` compared to using `||`.